pkg/install/generate_test.go

```diff
@@ -10,6 +10,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	testName      = "foo"
```

Comment:
These are both only used in one place. Can we use the string literals there rather than declaring constants?

Reply:
I was taught to avoid magic values but I don't even do this consistently. Will move them inline.

Reply:
I agree that magic values can be problematic. I'm not sure that applies the same to test code. I prefer to see the magic values right where they are being used for tests. That may be just me, though.